phpbu/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+- block:
```

Comment:
Not sure I like this style very much, but only aesthetically. No need to change if you like it

Reply:
I'm not a massive fan either. Do you think it would be better split into anther file and an include?